<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Factorial:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_facInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="pb_facStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_facPausRes">
        <property name="text">
         <string>Pause / Resume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_facAbort">
        <property name="text">
         <string>Abort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="te_facOutput"/>
      </item>
      <item>
       <widget class="QLabel" name="l_facStatus">
        <property name="text">
         <string>Status: Idle</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Prime numbers:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>From:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_primeInFrom"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>To:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_primeInTo"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pb_primeStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_primePausRes">
        <property name="text">
         <string>Pause / Resume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_primeAbort">
        <property name="text">
         <string>Abort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Results:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="te_primeOutput"/>
      </item>
      <item>
       <widget class="QLabel" name="l_primeStatus">
        <property name="text">
         <string>Status: Idle</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
